docs/backends/at.md

```diff
@@ -0,0 +1,18 @@
+# AT APDU Backend
+
+The AT Backend is used to communicate with the eUICC via AT commands over a serial interface.
+It is commonly used with cellular modems that support AT commands for SIM card management.
+
+## Unix-like platform
+
+If your needs rootless access to serial port
+
+1. <https://askubuntu.com/a/133244> (Debian/Ubuntu)
+2. <https://bbs.archlinux.org/viewtopic.php?id=178552> (Arch Linux)
```

Comment:
Use the official wiki instead. For example, https://wiki.archlinux.org/title/Working_with_the_serial_console#Connect_using_a_terminal_emulator_program for Arch Linux.